tests/test_bfh.py

```diff
@@ -244,6 +244,15 @@ def test_schema_can_use_defaults(self):
         s = DefaultsSchema(defaulted=None).serialize()
         self.assertEqual(s.get("defaulted"), "testing")
 
+    def test_schema_class_attribs_not_set(self):
+        class Myschema(Schema):
+            wow = IntegerField()
+
+        a = Myschema(wow=1)
+        b = Myschema()
+        self.assertEqual({"wow": 1}, a._raw_input)
```

Comment:
hmm, i prefer to test behavior, not implementation... perhaps we can cover the same thing using All transformation.